google-cloud-spanner/src/main/java/com/google/cloud/spanner/BackupInfo.java

```diff
@@ -28,8 +28,22 @@
 
     abstract Builder setSize(long size);
 
+    /**
+     * Output only.
+     *
+     * <p>The customer-manager encryption key version used to encrypt the backup.
```

Comment:
not the key itself but only the version of the key?

Reply:
Updated the comment to the proto comment.